Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/AssetBrowserBus.h

```diff
@@ -377,5 +377,23 @@ namespace AzToolsFramework
         };
         using AssetBrowserFileCreationNotificationBus = AZ::EBus<AssetBrowserFileCreationNotifications>;
 
+        //////////////////////////////////////////////////////////////////////////
+        // File action notifications
+        //////////////////////////////////////////////////////////////////////////
+
+        //! Used for sending and/or recieving notifications regarding source file manipulation through the Asset Browser.
+        class AssetBrowserFileActionNotifications
```

Comment:
I think this is OK for now, but I'd like to have a discussion on what this should look like long term. We'll probably have Asset Processor sending file manipulation events in the future, so I'm thinking we'll end up wanting to move this to a more central location that isn't as specific to the AssetBrowser?

Maybe a new "AssetStateMessages.h" added to:
https://github.com/o3de/o3de/tree/development/Code/Framework/AzToolsFramework/AzToolsFramework/Asset

Something that does not imply the originator of the message, I think listeners to this won't care if the relocation was done via the Asset Browser, Asset Processor, or externally on disk directly (which would probably route through the Asset Processor, which would detect and send the message)

Reply:
Yeah, this isn't necessarily an Asset Browser specific event. If/when Asset Processor supports file moves, it seems like it would make sense to add the event to the AssetSystemNotificationBus and refine SourceFileChanged.

For now, the AssetBrowserBus.h file already contained an existing AssetBrowserFileCreationNotificationBus which is similar, so I just followed that pattern.